modules/media/help/Setup.md

```diff
@@ -0,0 +1,28 @@
+## Media module set up 
+
+Set the `mediaPath` config value to the desired path on the server where the files 
+will be uploaded.
+
+Create the directory on the server.
+
+The upload path must be readable and writable by your web server; 
+either the web server `user` or `group` must have read and write permissions.
+
+The default group for your web server process is listed below:
+```
+Ubuntu: $GROUP$ = www-data
+CentOS: $GROUP$ = apache
+```
+
+To find the `user` of your web server, run:<br/> 
```

Comment:
@driusan i think this was merged prematurely. this file should be in markdown, and not have any html @victoriafoing